### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Heft9nic**

### CODE REVIEW

File: src/Acme/BlogBundle/Resources/views/Blog/stas.html.twig [hunk 1]
```diff
@@ -1,7 +1,42 @@
...
+{% endblock %}
  {% block body %}
+     <div id="Header">
      <h1>Hello {{ blog }}</h1>
+     <a href="{{ path('blog_show_user', {'slug':1}) }}">you</a>.
      <hr color="blue">
+     <ul id="nav">
+         <li id="thome"><a href="https://vk.com/stas.kirman">My page vk</a></li>
+         <li id="tguides"><a href="https://github.com/Heft9nic" class="selected">My page GitHub</a></li>
+         <li id="tmag"><a href="http://geekhub.ck.ua/">Page GeekHab</a></li>
```
Comment (alexandrzhulev) [inline src/Acme/BlogBundle/Resources/views/Blog/stas.html.twig:22]: GeekHub, not GeekHab

File: src/Acme/BlogBundle/Resources/views/Blog/stas.html.twig [hunk 2]
```diff
@@ -1,7 +1,42 @@
...
+
+     </ul>
+     </div>
+   <div id="footer">
+       <p>{{ time|date }}</p>
+   </div>
      {#<a href="{{ url('home') }}">Home</a>#}
+     {{ prise|capitalize|lower|title }}
+     {#{% for prise in prices %}#}
```
Comment (alexandrzhulev) [inline src/Acme/BlogBundle/Resources/views/Blog/stas.html.twig]: better to delete commented lines

File: src/Acme/BlogBundle/Controller/BlogController.php [hunk 3]
```diff
@@ -23,8 +23,11 @@ public function homepageAction($page)
...
     public  function stasAction()
     {
+               $time = time();
         return $this->render('AcmeBlogBundle:Blog:stas.html.twig', array(
             'blog' => 'stas',
+            'time' => $time,
         ));
     }
+
 }
```
Comment (alexandrzhulev) [inline src/Acme/BlogBundle/Controller/BlogController.php:33]: in this file - delete unnecessary empty lines between namespace and use (4, 5), delete commented lines (13, 14), use array as [ ]

File: src/Acme/BlogBundle/Resources/views/Blog/stas.html.twig (hunk 2 above)
Comment (midori-no-risu) [inline src/Acme/BlogBundle/Resources/views/Blog/stas.html.twig]: +1

File: app/Resources/views/base.html.twig [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title>{% block title %}Welcome!{% endblock %}</title>
+    {% block stylesheets %}{% endblock %}
+    <link rel="shortcut icon" href="{{ asset('favicon.ico') }}" />
+</head>
+<body>
+{% block body %}{% endblock %}
+{% block javascripts %}{% endblock %}
+</body>
+</html>
```
Comment (midori-no-risu) [inline app/Resources/views/base.html.twig:13]: new line

File: app/SymfonyStandard/Composer.php [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+use Composer\Script\CommandEvent;
+
+class Composer
+{
+    public static function hookRootPackageInstall(CommandEvent $event)
+    {
+        $event->getComposer()->getEventDispatcher()->addSubscriber(new RootPackageInstallSubscriber());
+    }
+}
```
Comment (midori-no-risu) [inline app/SymfonyStandard/Composer.php:22]: what is this for?

File: app/config/config_dev.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        #    level:  info
+        #chromephp:
+        #    type:   chromephp
+        #    level:  info
+
+assetic:
+    use_controller: "%use_assetic_controller%"
+
+#swiftmailer:
+#    delivery_address: me@example.com
```
Comment (midori-no-risu) [inline app/config/config_dev.yml:36]: why are these whole files in your pr?

File: app/config/routing_dev.yml [hunk 7]
```diff
@@ -0,0 +1,18 @@
+_wdt:
```
Comment (midori-no-risu) [inline app/config/routing_dev.yml:1]: super informative name

File: app/config/routing_dev.yml [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+    prefix:   /_profiler
+
+_configurator:
+    resource: "@SensioDistributionBundle/Resources/config/routing/webconfigurator.xml"
+    prefix:   /_configurator
+
+_main:
+    resource: routing.yml
+
+# AcmeDemoBundle routes (to be removed)
```
Comment (midori-no-risu) [inline app/config/routing_dev.yml:16]: then why not to remove?

**Comment by alexandrzhulev** [comment]
First resolve merge conflicts and fix previous bugs

**Pull request merged**